### PR TITLE
dev-vcs/git-lfs: Bump to 2.12.1

### PR DESCRIPTION
**Pull request by holgersson32644**
Package-Manager: Portage-3.0.7, Repoman-3.0.1
Signed-off-by: Nils Freydank <holgersson@posteo.de>

### CODE REVIEW

File: dev-vcs/git-lfs/git-lfs-2.12.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+RESTRICT+=" !test? ( test )"
+
+src_compile() {
+	set -- go build \
+		-ldflags="-X ${EGO_PN}/config.GitCommit=${GIT_COMMIT}" \
+		-mod vendor -v -work -x \
+		-o git-lfs git-lfs.go
+	echo "$@"
+	"$@" || die
```
Comment (juippis) [inline dev-vcs/git-lfs/git-lfs-2.12.0.ebuild:34]: What's the purpose of these `set`, `echo` etc? Can't you just call `go <string> || die`?

File: dev-vcs/git-lfs/git-lfs-2.12.0.ebuild [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+}
+
+src_test() {
+	set -- go test \
+		-ldflags="-X ${EGO_PN}/config.GitCommit=${GIT_COMMIT}" \
+		-mod vendor \
+		./...
+	echo "$@"
+	"$@" || die
+}
```
Comment (juippis) [inline dev-vcs/git-lfs/git-lfs-2.12.0.ebuild:54]: Here too. And ideally `src_test` comes after `src_compile`.

File: dev-vcs/git-lfs/git-lfs-2.12.0.ebuild (hunk 1 above)
Comment (holgersson32644) [reply dev-vcs/git-lfs/git-lfs-2.12.0.ebuild]: Fixed. I took this style from the older versions and didn't touch it as it worked.

File: dev-vcs/git-lfs/git-lfs-2.12.0.ebuild (hunk 2 above)
Comment (holgersson32644) [reply dev-vcs/git-lfs/git-lfs-2.12.0.ebuild]: sure

**Review by juippis — APPROVED** [review]

**Pull request closed without merging**